Find Food Trucks SF/Find Food Trucks SF/BookmarksViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  BookmarkViewController.swift
+//  Find Food Trucks SF
+//
+//  Created by Mark on 4/22/18.
+//  Copyright © 2018 TheFootGang. All rights reserved.
+//
+
+import UIKit
+
+class BookmarksViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+
+    @IBOutlet weak var tableView: UITableView!
+    @IBAction func unwindToBookmarksFromDetail(segue:UIStoryboardSegue) {}
+
+    var bookmarks: [FoodTruck] = []
+    let service: FoodTruckService = FoodTruckService()
+    let userDefaults = UserDefaults.standard
+    @IBAction func tapRefresh(_ sender: UIButton) {
+        
+        viewDidLoad()
+        for bookmarked in self.bookmarks {
+            print(bookmarked.name)
+        }
+        self.tableView.reloadData()
+    }
+    
+    override func viewDidLoad() {
+        self.bookmarks.removeAll()
+        super.viewDidLoad()
+        service.getFoodTrucks() { [unowned self] (foodTrucks: [FoodTruck]?) in
+            guard let trucks = foodTrucks else {
+                print("Error fetching food trucks.")
+                return
+            }
+            
```

Comment:
This loads the data of food trucks that are keyed and bookmarked.

Reply:
In the future, this might not be the best solution to store truck ids as keys, maybe look at core data.